src/modules/API/render.js

```diff
@@ -0,0 +1,64 @@
+import { getMeals, getLikes, postLikes } from './get';
+import showCounter from './counter';
+import commentsModel from '../comments/commentModel';
+import reservation from '../reservation/reservation';
+
+const list = document.querySelector('.cards-container');
+const bodyEl = document.querySelector('body');
+
+const renderLikes = () => {
+  getLikes().then((data) => {
+    data.forEach((element) => {
+      const likes = document.getElementById(
+        `likes_${element.item_id.replace('heart_', '')}`,
+      );
+      if (likes) {
+        likes.textContent = `${element.likes} Likes`;
+      }
+    });
+  });
+};
```

Comment:
- [ ] As required for the project all counters sections to be covered with unit tests using [the Jest Framwework](https://jestjs.io/).get covered with unit tests using [the Jest Framwework](https://jestjs.io/), kindly ensure that you also add tests on the `Likes_counter`

Reply:
Added changes requested